resources/install.rb

```diff
@@ -0,0 +1,161 @@
+property :name, kind_of: String, name_property: true
```

Comment:
Name property shouldn't be here. It comes for free

Reply:
okay will pull it out